code/modules/jobs/job_types/bos.dm

```diff
@@ -236,6 +236,8 @@ Head Scribe
 	belt = 			/obj/item/storage/belt/utility/full/engi
 	id = 			/obj/item/card/id/dogtag
 	backpack_contents = list(
+		/obj/item/gun/ballistic/automatic/pistol/pistol14 = 1,
+		/obj/item/ammo_box/magazine/m14mm = 2,
```

Comment:
You added this to the Head Scribe, not the Head Knight.

Reply:
🧠